Makefrag

```diff
@@ -13,7 +13,11 @@ space := $() $()
 splitConfigs := $(subst $(comma), ,$(CONFIG))
 configBases := $(foreach config,$(splitConfigs),$(lastword $(subst ., ,$(config))))
 CONFIG_STR := $(subst $(space),_,$(configBases))
+ifdef STABILIZE
+long_name = $(PROJECT).$(CONFIG_STR).stabilized
```

Comment:
is this just for debugging / comparison purposes?

Reply:
what I mean is, do you think we should merge the PR with this change?

Reply:
no this is just for debug/comparision

Reply:
It might make sense to leave some debug scripts etc. in, so in the future its easier to debug this type of problem. Perhaps we can make the debug stuff a bit more generic, like call it "NAME_DEBUG" or something like that? In general I like this kind of script because it helps us measure how 'stable' the module names are.